Issues with installing when the commander is 13.1.0

Hi guys,

We already use `commander@"13.1.0"` and faced some issues installing `extra-typings`:

```
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @commander-js/extra-typings@13.0.0
npm error Found: commander@13.1.0
npm error node_modules/commander
npm error   commander@"13.1.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peer commander@"~13.0.0" from @commander-js/extra-typings@13.0.0
npm error node_modules/@commander-js/extra-typings
npm error   @commander-js/extra-typings@"13.0.0" from the root project
npm error
npm error Conflicting peer dependency: commander@13.0.0
npm error node_modules/commander
npm error   peer commander@"~13.0.0" from @commander-js/extra-typings@13.0.0
npm error   node_modules/@commander-js/extra-typings
npm error     @commander-js/extra-typings@"13.0.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

The `--legacy-peer-deps` workaround works for us, but changing the CI configuration is a bit troublesome, and we have no other reason to do that. I'm not a JS/NPM expert but it is because of the peer dependency `"commander": "~13.0.0"` that rules out `13.1.0`, right? Would you mind having a look if this was intentional?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,13 @@ The version numbering does not follow semantic versioning but instead aligns wit
 <!-- markdownlint-disable MD024 -->
 <!-- markdownlint-disable MD004 -->
 
+## [13.1.0] (2025-01-25)
+
+## Changed
+
+- peer dependency on Commander 13.1.x
+- update dependencies
+
 ## [13.0.0] (2024-12-31)
 
 ### Added
@@ -188,6 +195,7 @@ Published from wrong branch.
 - inferred types for `.action()`
 - inferred types for `.opts()`
 
+[13.1.0]: https://github.com/commander-js/extra-typings/compare/v13.0.0...v13.1.0
 [13.0.0]: https://github.com/commander-js/extra-typings/compare/v12.1.0...v13.0.0
 [13.0.0-1]: https://github.com/commander-js/extra-typings/compare/v12.1.0...v13.0.0-1
 [12.1.0]: https://github.com/commander-js/extra-typings/compare/v12.0.1...v12.1.0

--- a/package.json
+++ b/package.json
@@ -43,14 +43,14 @@
   },
   "homepage": "https://github.com/commander-js/extra-typings#readme",
   "peerDependencies": {
-    "commander": "~13.0.0"
+    "commander": "~13.1.0"
   },
   "devDependencies": {
     "@jest/globals": "^29.3.1",
     "@types/node": "^22.10.1",
-    "commander": "^13.0.0",
+    "commander": "~13.1.0",
     "eslint": "^9.16.0",
-    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-prettier": "^10.0.1",
     "eslint-plugin-jest": "^28.9.0",
     "globals": "^15.0.0",
     "jest": "^29.3.1",

